rs/nns/handlers/root/impl/canister/canister.rs

```diff
@@ -94,13 +87,8 @@ ic_nervous_system_common_build_metadata::define_get_build_metadata_candid_method
 ///
 /// This must be an update, not a query, because an inter-canister call to the
 /// management canister is required.
-#[export_name = "canister_update canister_status"]
-fn canister_status() {
-    over_async(candid_one, canister_status_)
-}
-
-#[candid_method(update, rename = "canister_status")]
-async fn canister_status_(canister_id_record: CanisterIdRecord) -> CanisterStatusResult {
+#[update]
```

Comment:
Why not query? 

NCR for this PR

Reply:
I think b/c queries can't call other canisters.